schemas/cloud.yml

```diff
@@ -17,8 +18,9 @@
       level: extended
       example: ec2
```

Comment:
@webmat Sorry I just saw this PR for some reason... Thanks for making the changes! I just found this `ec2` still in provider example. This probably should be `aws` instead.

Reply:
Oh jeez, we fixed the example in the text, but not in the `example` field, haha! Thanks for pointing it out 👍